### PR TITLE
Enable debugMetrics only for CI builds

### PR DESCRIPTION
**Pull request by binuinbaraj**

### CODE REVIEW

File: examples/pxScene2d/src/CMakeLists.txt [hunk 1]
```diff
@@ -45,6 +45,7 @@ option(BUILD_PXSCENE_STATIC_LIB "BUILD_PXSCENE_STATIC_LIB" OFF)
 option(BUILD_PXWAYLAND_SHARED_LIB "BUILD_PXWAYLAND_SHARED_LIB" OFF)
 option(BUILD_PXWAYLAND_STATIC_LIB "BUILD_PXWAYLAND_STATIC_LIB" ON)
 option(BUILD_PXSCENE_WITH_NEXUS_SUPPORT "BUILD_PXSCENE_WITH_NEXUS_SUPPORT" OFF)
+option(ENABLE_DEBUG_METRICS "ENABLE_DEBUG_METRICS" OFF)
```
Comment (mfiess) [inline examples/pxScene2d/src/CMakeLists.txt]: Please rename option from "ENABLE_DEBUG_METRICS" to "BUILD_DEBUG_METRICS".  The add the following block of code after this option:

if (BUILD_DEBUG_METRICS)
  message("Building with debug metrics")
  add_definitions(-DENABLE_DEBUG_METRICS)
endif (BUILD_DEBUG_METRICS)

File: ci/build_linux.sh [hunk 2]
```diff
@@ -19,7 +19,7 @@ cd temp
 if [ "$TRAVIS_PULL_REQUEST" = "false" ]
 then
 echo "***************************** Generating config files ****" >> $BUILDLOGS
-cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON .. >>$BUILDLOGS 2>&1;
+cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DENABLE_DEBUG_METRICS=ON .. >>$BUILDLOGS 2>&1;
```
Comment (mfiess) [inline ci/build_linux.sh]: Please rename "-DENABLE_DEBUG_METRICS=ON" to "-DBUILD_DEBUG_METRICS=ON".  Please see comment in CMakeLists.txt for more details.

File: ci/build_linux.sh [hunk 3]
```diff
@@ -28,7 +28,7 @@ checkError $? "cmake build failed for pxcore or rtcore" "Compilation error" "Che
 
 else
 echo "***************************** Generating config files ****"
-cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON .. 1>>$BUILDLOGS;
+cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DENABLE_DEBUG_METRICS=ON .. 1>>$BUILDLOGS;
```
Comment (mfiess) [inline ci/build_linux.sh]: Please rename "-DENABLE_DEBUG_METRICS=ON" to "-DBUILD_DEBUG_METRICS=ON".  Please see comment in CMakeLists.txt for more details.

**Comment by binuinbaraj** [comment]
https://github.com/pxscene/pxCore/pull/602 has updated changes as per review.

**Pull request closed without merging**